teps/0091-trusted-resources.md

```diff
@@ -298,6 +301,12 @@ spec:
        name: example-task2
        keyref: keypath2  # this defines the public key of this taskref
 ```
+`keyref` defines the path of public key of this taskRef. All the tasks within a pipeline need to be signed off, and they may come from different sources with different public keys.
+So a `keyref` is needed when signing the pipeline. The signing cli doesn't need to verify the referred keys when signing pipeline.
+
+In this TEP we proposed to choose option 1, for option 2 it would cause a bad user experience that if one pipeline contains a bunch of tasks, then users will need to configure them manually one by one. This would be a big cost for using this feature.
```

Comment:
I'm fine with making an "out-of-scope" declaration in this `TEP`, but we definitely will want to be able to support multiple keys in the future.

CUJ: "I a cluster admin, and I allow `Pipeline` creation for my application developers only of pipelines that use any of the following list of keys."

Reply:
Option 1 also supports multiple keys, I will clarify it in this part！